view/frontend/web/js/view/payment/method-renderer/omise-offsite-installment-method.js

```diff
@@ -56,19 +58,120 @@ define(
                 return this;
             },
 
+            /**
+             * Format Price
+             * 
+             * @param {float} amount - Amount to be formatted
+             * @return {string}
+             */
+            getFormattedAmount: function (amount) {
+                return priceUtils.formatPrice(amount, quote.getPriceFormat());
+            },
+
+            /**
+             * Get Installment minimum
+             * this function respects info from: https://www.omise.co/installment-payment
+             *
+             * NOTE: in the future this function should return data from capabilities object.
+             *
+             * @param {string} id - Bank ID
+             * @return {integer}
+             */
+            getInstallmentMinimum: function (id) {
+                switch (id) {
+                    case 'kbank': return 500;
+                    case 'bbl': return 500;
+                    case 'bay': return 300;
+                    case 'first_choice': return 300;
+                    case 'ktc': return 300;
+                }
+            },
+
+            /**
+             * Get Installment monthly interest rate
+             *
+             * NOTE: in the future this function should return data from capabilities object.
+             * 
+             * @param {string} id - Bank id
+             * @return {float}
+             */
+            getInstallmentInterestRate: function (id) {
+                switch (id) {
+                    case 'kbank': return 0.0065;
```

Comment:
Feels wrong to have these hardcoded in JS - wouldn't a better place be a config file?

Reply:
it will come from capabilities api, entire object is available here in js.

Reply:
These are not in the new capabilities API, and there is no plan to put them there AFAIK

Reply:
it needs to come from capabilities it is a matter of discussion with backend team.

changing plugins when those conditions will change it is huge pain.

also those companies have some promotions, that we will have easy to manage when it will come from capabilities.